### PR TITLE
Add/Edit Card, Delete Button on Card Detail

### PR DESCRIPTION
**Pull request by eezstreet**
Adds the ability to add/edit cards, and delete cards. If this is merged, this will finish issues #17 and #19.

The associated Firebase interactions need to be verified to see if they're working correctly, as the Add Card interaction wasn't working properly for me.

### CODE REVIEW

**Comment by pr4h6n** [comment]
You were incorrectly using the Card functions so it wasn’t pushing to Firebase properly

**Comment by pr4h6n** [comment]
I changed your code and it adds and deleted correctly. Haven’t tested edit though

File: app/src/main/java/com/example/app/baseballmessenger/AddEditCardActivity.java [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+        }
+
+        // If we're editing a card, map the data in the card to the entered fields
+        if(data != null)
+        {
+            // TODO: fix the image
+            saveCardButton.setText(saveChanges);
+
+            nameEntry.setText(data.name);
+            ownerEntry.setText(data.owner);
```
Comment (pr4h6n) [inline app/src/main/java/com/example/app/baseballmessenger/AddEditCardActivity.java]: User should not be able to edit the owner field.

**Comment by eezstreet** [comment]
How is the owner meant to be set?

________________________________

**Comment by pr4h6n** [comment]
Owner is set by the app as the current user's uuid

**Comment by eezstreet** [comment]
Seems to crash for me, on this line:

`Card c = new Card("", Handoff.currentUser.uuid,"","", 0, "", "", 0.0, 0, "", true, false);`

Handoff.currentUser is `null`. Could be a difference in hardware causing the crash? I'm on a Samsung Galaxy S5 Active.

**Review by pr4h6n — APPROVED** [review]

**Pull request merged**